doc/version_history.rst

```diff
@@ -6,6 +6,11 @@
 Version History
 ###############
 
+v1.3.0
```

Comment:
are you sure this needs to be a minor release? I would probably make is a patch instead, e.g. v1.2.2.

Reply:
Agree. Changed it to 1.2.2